frontend/apps/filemanager/filemanager.lua

```diff
@@ -134,7 +134,7 @@ function FileManager:setupLayout()
         height = icon_size,
         padding = Size.padding.default,
         padding_left = Size.padding.large,
-        padding_right = Size.padding.large,
+        padding_right = Size.padding.default,
```

Comment:
That's for the cross icon to align with the top right corner ?
Feels like a adhoc hack :) Although it looks logical now: it is the same value as padding_top - so if the icon itself is centered, sticking it to the corner, with the same padding on both touching edges, should have it centered.
It it still OK with the Plus icon ? Shouldn't the Home icon on the left have this as its padding_left for balance?